demo/index.html

```diff
@@ -128,11 +129,10 @@
   }});
 </script>
 <script src="/node_modules/rxjs/bundles/Rx.js"></script>
-<script src="/node_modules/angular2/bundles/angular2.js"></script>
 <script src="../bundles/ng2-formly.js"></script>
```

Comment:
As mention earlier demo was pointing to angular2 instead of @angular. The clean up of node_module and npm install again will help resolve any problem.